wwwroot/data/regionMapping.json

```diff
@@ -307,13 +307,36 @@
             "regionProp": "ELECT_DIV",
             "textCodes": true,
             "aliases": [
-                "divisionnm",
-                "ced_aec",
                 "ced_aec_2011"
             ],
             "description": "Commonwealth electoral districts (Australian Electoral Commission)",
             "regionIdsFile": "data/regionids/region_map-FID_COM20111216_ELB_region_ELECT_DIV.json"
         },
+        "CED_AEC_2016": {
+            "layerName": "region_map:FID_COM20160509_ELB",
+            "server": "http://geoserver.nationalmap.nicta.com.au/region_map/ows",
+            "regionProp": "DIV_ID",
```

Comment:
Ok so the 2016 CEDs use a different region property, and this is the thing that must be put in the column of a CSV file, right?  And also we've changed `ced_aec` to refer to the 2016 version.  So isn't that a breaking change?  Previously you could have a CSV file this:

```
ced_aec,value
Flynn,1
```

but now that won't load, you instead need to do:

```
ced_aec,value
311,1
```

Thoughts on that @tobybellwood?